workflow/controller/workflowpod_test.go

```diff
@@ -61,7 +61,7 @@ script:
 func TestScriptTemplateWithVolume(t *testing.T) {
 	tmpl := unmarshalTemplate(scriptTemplateWithInputArtifact)
 	woc := newWoc()
-	err := woc.executeScript(tmpl.Name, tmpl, "")
```

Comment:
The argument of `orgTmpl` is for an original template which is the one before resolving references. We can just pass `tmpl` instead of `wfv1.Template{}` and it's better to simulate real use of these methods.. Honestly speaking, I didn't want to allow passing arbitrary `orgTmpl` and that's one reason why I put the initialization logic in `executeTemplate` and passed actual `node` to these methods.